mp/src/game/server/momentum/mom_timer.cpp

```diff
@@ -398,6 +398,12 @@ CON_COMMAND_F(mom_restart_stage,
         track = pPlayer->m_Data.m_iCurrentTrack;
     }
 
+    if (pPlayer->m_iLinearTracks.Get(track))
```

Comment:
This should be done before the safeguard check on line 386. It would otherwise be safeguarding a dud command.

Reply:
And I see it gets the track from the player, so really to fix this just move the safeguard down below this check, but before the actual teleport command.